crates/driver/src/domain/eth/eip712.rs

```diff
@@ -7,25 +7,3 @@
 /// https://eips.ethereum.org/EIPS/eip-712#definition-of-domainseparator
 #[derive(Debug, Clone, Copy)]
 pub struct DomainSeparator(pub [u8; 32]);
-
-#[derive(Debug)]
-pub struct DomainFields {
-    pub type_hash: &'static [u8],
-    pub name: &'static [u8],
-    pub version: &'static [u8],
-    pub chain_id: chain::Id,
-    pub verifying_contract: super::ContractAddress,
-}
-
-impl DomainSeparator {
-    pub fn new(fields: &DomainFields) -> Self {
-        let abi_string = ethabi::encode(&[
-            ethabi::Token::Uint(web3::signing::keccak256(fields.type_hash).into()),
-            ethabi::Token::Uint(web3::signing::keccak256(fields.name).into()),
-            ethabi::Token::Uint(web3::signing::keccak256(fields.version).into()),
-            ethabi::Token::Uint(ethereum_types::U256::from(u64::from(fields.chain_id))),
-            ethabi::Token::Address(fields.verifying_contract.into()),
-        ]);
-        Self(web3::signing::keccak256(abi_string.as_slice()))
-    }
-}
```

Comment:
Just to double-check, this is not related to this PR and just removes previously unused code, right?

Reply:
True. I noticed it because the change touches `chain_id`